launch/tier4_map_launch/config/lanelet2_map_loader.param.yaml

```diff
@@ -0,0 +1,7 @@
+/**:
```

Comment:
No longer need for this config file, since `tier4_map_launch/config` directory has been removed (https://github.com/autowarefoundation/autoware.universe/pull/2722).